src/pages/invitee/EventInvite.jsx

```diff
@@ -20,24 +27,46 @@ const EventInvite = () => {
       ...inviteDetails,
       [name]: value
     })
-  }
+  };
+
+  const getEventDetails =  async () => {
+    const token = localStorage.getItem("jwt-token");
+    const requestEventDetails = {
+      headers: { 
+        'Content-Type': 'application/json',
+        'Authorization': `Bearer ${token}`
+      },
+    }
+
+    fetch(`https://prybar.onrender.com/api/v1/event/token/${eventToken}`, requestEventDetails)
+    .then(response => response.json())
+    .then(result => setEventData(result))
+  };
 
   const addParticipant = (e) => {
     e.preventDefault();
     const requestOptions = {
       method: 'POST',
       headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify({...inviteDetails, event_id: eventId})
+      body: JSON.stringify({...inviteDetails, event_id: eventData.data._id})
     }
+
     fetch('https://prybar.onrender.com/api/v1/participant/addpart', requestOptions)
-    .then(response => response.json())
+    .then(response => console.log(response.json()))
     .then((result) => {
       if(result.status === "success"){
         setTimeout(() => {
+          setResultMsg(result?.message)
           navigate('/event_invite/event_invite_response')
         }, 2000)
+      } else {
+        setResultMsg(result?.message)
       }
     })
+    .catch(error => {
```

Comment:
- [ ] I will recommend you separate your logic. You can follow up on whats being done in the services folder. 

from rootEndpoint,js-> userServices.js and finally importing your request here which will make your code cleaner. Meanwhile we will be changing the routes soon, so having a central place to hold our API request is very vital